README.md

```diff
@@ -18,7 +18,120 @@ to produce the equivalent mercurial commits to the right repo (depending on the
 It finally pushes the changes to the remote mercurial repository… then goes back to step one to
 process the next message in the queue.
 
-## build and test
+## Run
+
+```console
+$ docker compose up -d
+```
+
+The logs from the syncer can be displayed with
+
+```console
+$ docker compose logs -f sync
+```
+
+Repositories can be seen on the host in the `clones` subdirectory. Remember to delete if you want to start fresh.
+
+```
+$ ls clones
+mozilla-beta  mozilla-esr115  mozilla-esr128  test-repo-git  test-repo-hg
+```
+
+See the `create_clones.sh` for the creation logic. `test-repo-hg` is the original HG repository (but is otherwise unused), `test-repo-git` is the source Git repository, while the various `mozilla-*` are the targets HG repositories to sync to. The syncer will also create its own `firefox-releases` Git repo to work in.
```

Comment:
nit
```suggestion
See the `create_clones.sh` for the creation logic. `test-repo-hg` is the original Mercurial repository (but is otherwise unused), `test-repo-git` is the source Git repository, while the various `mozilla-*` are the targets Mercurial repositories to sync to. The syncer will also create its own `firefox-releases` Git repo to work in.
```

I think `Hg` also works.